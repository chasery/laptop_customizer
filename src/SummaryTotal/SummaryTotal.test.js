import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import SummaryTotal from "./SummaryTotal";

describe("SummaryTotal component", () => {
  it("renders without errors", () => {
    const wrapper = shallow(<SummaryTotal />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
