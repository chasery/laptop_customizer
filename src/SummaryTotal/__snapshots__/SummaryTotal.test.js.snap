// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SummaryTotal component renders without errors 1`] = `
<div
  className="summary__total"
>
  <div
    className="summary__total__label"
  >
    Total
  </div>
  <div
    className="summary__total__value"
  >
    $0.00
  </div>
</div>
`;
